firstapp/chrome.xml

```diff
@@ -0,0 +1 @@
nds="[0,98][1440,322]"><node index="0" text="" resource-id="com.android.chrome:id/toolbar" class="android.widget.FrameLayout" package="com.android.chrome" content-desc="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" scrollable="false" long-clickable="false" password="false" selected="false" bounds="[0,98][1440,294]"><node index="0" text="" resource-id="com.android.chrome:id/home_button" class="android.widget.ImageButton" package="com.android.chrome" content-desc="Home" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" scrollable="false" long-clickable="true" password="false" selected="false" bounds="[0,98][168,294]" /><node index="2" text="" resource-id="com.android.chrome:id/toolbar_buttons" class="android.widget.LinearLayout" package="com.android.chrome" content-desc="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" scrollable="false" long-clickable="false" password="false" selected="false" bounds="[1104,98][1440,294]"><node index="0" text="" resource-id="com.android.chrome:id/tab_switcher_button" class="android.widget.ImageButton" package="com.android.chrome" content-desc="1 open tab, tap to switch tabs" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" scrollable="false" long-clickable="true" password="false" selected="false" bounds="[1104,98][1272,294]" /><node index="1" text="" resource-id="com.android.chrome:id/menu_button_wrapper" class="android.widget.FrameLayout" package="com.android.chrome" content-desc="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" scrollable="false" long-clickable="false" password="false" selected="false" bounds="[1272,98][1440,294]"><node index="0" text="" resource-id="com.android.chrome:id/menu_button" class="android.widget.ImageButton" package="com.android.chrome" content-desc="More options" checkable="false" checked="false" clickable="true" enabled="true" focusable="true" focused="false" scrollable="false" long-clickable="false" password="false" selected="false" bounds="[1272,98][1440,294]" /></node></node></node></node></node></node></node></node></node></node><node index="1" text="" resource-id="android:id/statusBarBackground" class="android.view.View" package="com.android.chrome" content-desc="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" scrollable="false" long-clickable="false" password="false" selected="false" bounds="[0,0][1440,98]" /><node index="2" text="" resource-id="android:id/navigationBarBackground" class="android.view.View" package="com.android.chrome" content-desc="" checkable="false" checked="false" clickable="false" enabled="true" focusable="false" focused="false" scrollable="false" long-clickable="false" password="false" selected="false" bounds="[0,0][0,0]" /></node></hierarchy>
```

Comment:
What is the purpose of chrome.xml?  And why is it formatted all in on line?

Reply:
oops-- that's a leftover from an old experiment